introduction.md

```diff
@@ -0,0 +1 @@
+this isn't patrick. loving the Rails path at CodeSchool. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.